public/docs/ts/latest/cookbook/a11y.jade

```diff
@@ -0,0 +1,444 @@
+    :marked
+      In `ARIA` we refer to the `aria-...` attributes as `ARIA States` or `ARIA Properties`. The difference between
+      the two should become clear as you progress through this cookbook. However, `ARIA Properties` are not 
+      actual `HTML` element properties but decorating attributes refering to properties. When we refer to setting an 
+      `ARIA Property` in the code you will **HAVE** to do
+      it with an Angular&nbsp;2 attribute binding. This is simply a terminology clash.
+
+.l-main-section
+:marked
+  ## Native elements vs. custom components
+
+  Angular&nbsp;2 gives you a lot of power at your fingertips to create extremely powerful components. But spare a
+  thought for the native `HTML` elements before you decide to build something new.
+
+  The makers of the browsers have spent a lot of time thinking about the very same issue that has brought
+  you to this page today and have provided you with a lot of functionality out-of-the-box when you make use of
+  native `HTML` elements.
+
+  We would like to suggest the following rule of thumb when building your applications:
+
+.l-sub-section
+  :marked
+    If there is already a native element inside `HTML` providing the function that you need, use
+    that element instead of writing your own.
+
+:marked
+  So, for example, if you want to accept user text input, use the `input` element instead of contructing something new.
+
+  This way you have to think less about things like focus management, tabindex, etc. and have more time to think about
+  your code.
+
+  But, we know this is not always possible, so this guide will also show you what you can do to make your own
+  components as accessible as possible.
+
+  So without further ado, let us see how easy it is to get big `a11y` wins in our applications!
+
+.l-main-section
+:marked
+  ## Important note on styling in this chapter
+
+.callout.is-important
+  header Angular Pages Do Not Require A Style Library
+  :marked
+    Angular makes no use of the `container`, `row`, `col-xs-12`, `checkbox`, `radio`, `form-group`, `form-control`,
+    and `btn` classes or the styles of any external library. Angular apps can use any CSS library
+    ... or none at all.
+    
+:marked
+  The `container`, `row`, `col-xs-12`, `checkbox`, `radio`, `form-group`, `form-control`, and `btn` classes
+  come from [Twitter Boostrap](http://getbootstrap.com/css/). This is a purely cosmetic addition to prettify
+  the examples so that they are as visually appealing as they are accessible.
+
+  But always remember, just because something looks good, it does not mean that it is accessible.
+
+  The good news is, here we strive to achieve both!
+  
+.l-main-section
+<a id="toc"></a>
+:marked
+  ## Table of contents
+
+  [Accessible form control labels](#form-control-labels)
+
+  [Using keyboard shortcuts](#keyboard-shortcuts)
+
+  [Component that knows its own role](#component-roles)
+
+  [Managing focus](#managin-focus)
```

Comment:
Missing a `g` here in the link, I think it won't link you to that section currently